examples/webpack/package.json

```diff
@@ -5,8 +5,8 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "webpack": "^4.10.2",
-    "webpack-cli": "^2.1.4",
+    "webpack": "^4.20.2",
```

Comment:
As outlined in #10177, recent Webpack version are breaking the PDF.js library.
To prevent issues, please pin it to the same *exact* version as in the main `package.json` file: https://github.com/mozilla/pdf.js/blob/2194aef03e155c3ee0f2f1491132b5b4e6dd6cfd/package.json#L40